cron/monthly/user-report.js

```diff
@@ -112,7 +112,10 @@ const processBacker = async FromCollectiveId => {
   const query = {
     attributes: ['CollectiveId', 'HostCollectiveId'],
     where: {
-      FromCollectiveId,
+      [Op.or]: {
+        FromCollectiveId,
+        UsingVirtualCardFromCollectiveId: FromCollectiveId,
+      },
```

Comment:
No need to support Virtual Card here at the moment.

Reply:
Yes sorry @marcogbarcellos, I didn't realized this was user report and user cannot emit virtualcards, my first comment was misleading.